packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Models/Types/EnumTypeValue.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp.Input;
+
+namespace Microsoft.Generator.CSharp
+{
+    public class EnumTypeValue
+    {
+        public EnumTypeValue(InputEnumTypeValue value)
```

Comment:
in the future, we might need this type to take in the `SourceInputModel` as well to take consider of customization code changes.
But now we only need the value.
Also since the `EnumTypeProvider` should have a public property of `Values` documenting the values it carries, we should not use the input type `InputEnumValue` just because the `EnumTypeProvider` is an output class, and the values it carries should also be output classes.